configs/nextflow.hopper.config

```diff
@@ -0,0 +1,170 @@
+// Hopper specific configuration for micro wgs pipeline //
+
+singularity{
+  enabled = true
+  runOptions = '--bind /fs1/ --bind /local/'
```

Comment:
also needs to bind fs2

Reply:
Do you want the results to be written to `fs2`? Other than that I'm not sure it's necessary or?

Reply:
It will read data from fs2?